src/cider/nrepl/middleware/info.clj

```diff
@@ -40,53 +39,28 @@
                  (when-let [forms (:forms info)]
                    {:forms-str (forms-join forms)})
                  (when-let [file (:file info)]
-                   (clj-info/file-info file))
+                   (info/file-info file))
                  (when-let [path (:javadoc info)]
-                   (clj-info/javadoc-info path)))
+                   (info/javadoc-info path)))
           format-nested
           blacklist
           u/transform-value))))
 
-(defn info-cljs
-  [env symbol ns]
-  (some-> (cljs-info/info env symbol ns)
-          (select-keys [:file :line :ns :doc :column :name :arglists])
-          (update
-           :file
-           (fn [f]
-             (if (System/getProperty "fake.class.path")
```

Comment:
@bbatsov Are you referring to this code to go away? Or is it something else?

Reply:
Yep.